channels/OFF-CHAIN.md

```diff
@@ -316,33 +399,48 @@ initiator to send their signature to the responder.
 
 *Responder*:
 
+- MUST abort if the size of the payload does not match `length`
+- MUST abort if the `data` cannot be deserialized into a valid
+  `channel_create_tx` object
 - MUST abort if the signature is invalid for the provided transaction data
+- SHOULD abort if the `round` of the state is 
```

Comment:
Probably missing the part about the round being [> 1](https://github.com/aeternity/epoch/blob/master/apps/aechannel/src/aesc_create_tx.erl#L319)